lib/rspec/support/ruby_features.rb

```diff
@@ -147,7 +146,6 @@ def supports_rebinding_module_methods?
             false
           end
         end
-        # rubocop:enable Lint/Eval
```

Comment:
Does rubocop no longer support this way of disabling checks?  Or do you just prefer putting the exception in the config file?

Reply:
it does, but it was redundant here